geo/src/algorithm/bool_ops/mod.rs

```diff
@@ -2,6 +2,11 @@ mod i_overlay_integration;
 #[cfg(test)]
 mod tests;
 
+#[cfg(feature = "multithreading")]
+use rayon::prelude::IntoParallelIterator;
+#[cfg(feature = "multithreading")]
+use rayon::prelude::*;
```

Comment:
These look redundant? If you want to avoid the prelude, importing `IntoParallelIterator` and `ParallelIterator` might suffice.

Reply:
Oops yep